content/en/docs/collector/internal-telemetry.md

```diff
@@ -113,3 +118,148 @@ journalctl | grep otelcol | grep Error
 ```
 
 {{% /tab %}} {{< /tabpane >}}
+
+## Types of internal observability
+
+The OpenTelemetry Collector aims to be an exemplar of observable service by
+clearly exposing its own operational metrics. It also collects host resource
+metrics that can help you understand if problems are caused by a different
+process on the same host. In this section, you can explore the different types
+of observability emitted by the Collector itself.
+
+### Values observable with internal metrics
+
+The Collector emits internal metrics for the following **current values**:
+
+- Resource consumption, including CPU, memory, and I/O.
+
+- Data reception rate, broken down by receiver.
+
+- Data export rate, broken down by exporters.
+
+- Data drop rate due to throttling, broken down by data type.
+
+- Data drop rate due to invalid data received, broken down by data type.
+
+- Throttling state, including Not Throttled, Throttled by Downstream, and
+  Internally Saturated.
+
+- Incoming connection count, broken down by receiver.
+
+- Incoming connection rate showing new connections per second, broken down by
+  receiver.
+
+- In-memory queue size in bytes and in units.
+
+- Persistent queue size.
+
+- End-to-end latency from receiver input to exporter output.
+
+- Latency broken down by pipeline elements, including exporter network roundtrip
+  latency for request/response protocols.
+
+{{% alert title="Important" color="warning" %}}
+
+- Rate values reflect the average rate of the last 10 seconds. They are exposed
+  in bytes/sec and units/sec (for example, spans/sec).
+- Measurements in bytes might be difficult and expensive to obtain. They should
+  be used with caution.
+
+{{% /alert %}}
+
+The Collector also emits internal metrics for these **cumulative values**:
+
+- Total received data, broken down by receivers.
+
+- Total exported data, broken down by exporters.
+
+- Total dropped data due to throttling, broken down by data type.
+
+- Total dropped data due to invalid data received, broken down by data type.
+
+- Total incoming connection count, broken down by receiver.
+
+- Uptime since start.
+
+### List of internal metrics
+
+The following table identifies each internal metric by name and description.
+Each metric is also categorized by instrumentation type and level of verbosity.
+
+| Metric name                                       | Description                                                                               | Type      | Level      |
```

Comment:
As you can see in the [preview](https://deploy-preview-4322--opentelemetry.netlify.app/), the table is a bit wide and might require scrolling in a narrow window. Does this concern us? If so, I have three thoughts on how to solve it.

1. I can remove the Type column from the table. That information is a nice-to-have, but it is available elsewhere.
2. I can remove the Level column and instead create tabbed panes in the [Configure internal metrics](https://opentelemetry.io/docs/collector/internal-telemetry/#configure-internal-metrics) section: a pane for Basic, Normal, Detailed that lists the corresponding metric names. 
3. I can do both 1 and 2.

WDYT?

Reply:
I like idea (2), can we try this first before dropping.

You might also consider to introduce line breaks for the long ones (via adding `<br>`), markdownlint will add a warning for that but not a breaking one.

Reply:
I've added both the tabbed panes to the earlier section and line breaks to the table. Both data sets are still incomplete until I get confirmation of verbosity levels from the Collector folks, but in the meantime, let me know what you think! :)

Reply:
This looks good, thanks! I wonder if we have something in our pocket for column sorting, cc @chalin